### PR TITLE
Added functionality for linking to other posts

### PR DESCRIPTION
**Pull request by jindaln**

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/InsertRepDatastoreServlet.java [hunk 1]
```diff
@@ -58,12 +58,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
...
-                //Prepend the Other tag with repName for uniqueness and 
-                // remove spaces in repName with replaceAll
-                Arrays.asList(repName.replaceAll("\\s+","") + "Other"), 
-                Arrays.asList(OTHER_TAB_PLATFORM));
+            List<Long> tabIds;
+            //Initializes rep's tab list with an Other tab prepended by repName for 
+            //uniqueness and remove spaces in repName with replaceAll
+            try {
+                tabIds = DatastoreManager.insertTabsInDatastore( 
+                    Arrays.asList(repName.replaceAll("\\s+","") + "Other"),
```
Comment (v1shanker) [inline src/main/java/com/google/sps/servlets/InsertRepDatastoreServlet.java:66]: keep the comment?

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -335,11 +345,28 @@ function displayRepAnswer(post, repName) {
...
-        postElement.appendChild(repAnswer);
+        displayCommentWithLinking(postElement, repAnswer, answer);
     }
 }
 
+//Displays comment and links to other posts if applicable
+function displayCommentWithLinking(postElement, container, commentObject) {
+    container.innerHTML = commentObject.name + ": ";
+    var wordsInComment = commentObject.comment.split(" ");
+    var regex = /@\d{16}/g;
```
Comment (v1shanker) [inline src/main/webapp/script.js:356]: comment to explain what this is?

File: src/main/webapp/script.js [hunk 3]
```diff
@@ -658,3 +685,9 @@ async function displayPoliticianPage(imgUrl) {
 
     returnToFeed(repName, document.getElementById("feed"));
 }
+
+//Logs out a user or rep
+function resetLocalStorage() {
```
Comment (epoms) [inline src/main/webapp/script.js:690]: Some changes here are also in another PR. Remember to Keep your workspace and PRs focused. If PRs are not getting approved fast enough, then let us know.

File: src/main/java/com/google/sps/servlets/InsertRepDatastoreServlet.java (hunk 1 above)
Comment (jindaln) [reply src/main/java/com/google/sps/servlets/InsertRepDatastoreServlet.java:66]: I just condensed them cause it was getting repetitive

**Pull request merged**